collector/netframework_clrexceptions.go

```diff
@@ -0,0 +1,132 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRExceptions
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRExceptions class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrexceptions"] = NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector
+}
+
+// A Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRExceptions metrics
+type Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector struct {
```

Comment:
You can leave out the `Win32_PerfRawData_` bit in the beginning.

Reply:
yes ok done